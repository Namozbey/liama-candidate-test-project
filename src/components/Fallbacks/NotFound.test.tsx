import React from "react";
import { render } from "@testing-library/react";
import { NotFound } from ".";

describe("Testing NotFound", () => {
  it("Snapshots", () => {
    const { asFragment } = render(<NotFound />);
    expect(asFragment()).toMatchSnapshot();
  });
});
