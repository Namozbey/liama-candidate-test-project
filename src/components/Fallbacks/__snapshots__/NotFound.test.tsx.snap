// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing NotFound Snapshots 1`] = `
<DocumentFragment>
  <div
    class="sc-bczRLJ fxlkcZ"
  >
    <img
      alt="not found image (404)"
      src=""
    />
  </div>
</DocumentFragment>
`;
